examples/wasm/push.rs

```diff
@@ -0,0 +1,36 @@
+use oci_distribution::{
+    client::{Config, ImageLayer},
+    manifest,
+    secrets::RegistryAuth,
+    Client, Reference,
+};
+use tracing::info;
+
+pub(crate) async fn push_wasm(
+    client: &mut Client,
+    auth: &RegistryAuth,
+    reference: &Reference,
+    module: &str,
+) {
+    info!(?reference, ?module, "pushing wasm module");
+
+    let data = std::fs::read(module).expect("Cannot read Wasm module from disk");
```

Comment:
Nit (because this is just an example): This is blocking IO in an async function. An async compatible read should be used instead

Reply:
Good point, fixed